bower.json

```diff
@@ -20,9 +20,8 @@
   "homepage": "https://github.com/PolymerElements/paper-radio-group",
   "ignore": [],
   "dependencies": {
-    "iron-a11y-keys-behavior": "PolymerElements/iron-a11y-keys-behavior#^1.0.0",
-    "iron-selector": "PolymerElements/iron-selector#^1.0.0",
-    "polymer": "Polymer/polymer#^1.1.0"
+    "polymer": "Polymer/polymer#^1.1.0",
+    "iron-menu-behavior": "PolymerElements/iron-menu-behavior#^1.1.7"
```

Comment:
Hrm, this is just what bower put there when I ran `bower install --save PolymerElements/iron-menu-behavior`... I don't think it should matter? The bigger issue might be that this relies on changes to IronSelector that aren't necessarily transitively 'included' by IronMenuBehavior's version. :/